import-service/handlers/importFileParser.js

```diff
@@ -6,51 +6,68 @@ const importFileParser = async (event) => {
   console.log(event);
   const s3 = new AWS.S3({region: 'us-east-1'});
   const BUCKET = process.env.CSV_BUCKET;
+  const sqs = new AWS.SQS({region: 'us-east-1'});
+  const sqsParams = {
+    DelaySeconds: 10,
+    MessageAttributes: {
+      'Count': {
+        DataType: 'String',
+        StringValue: 'empty'
+      }
+    },
+    MessageBody: '',
+    QueueUrl: 'https://sqs.us-east-1.amazonaws.com/489669634691/catalogItemsQueue'
+  };
 
   try {
-    const results = [];
-
     for (const record of event.Records) {
       const RECORD_KEY = record.s3.object.key;
-      const params = {
+      const bucketParams = {
         Bucket: BUCKET, Key: RECORD_KEY
-      }
-      const s3object = await s3.getObject(params).promise();
+      };
+      const s3object = await s3.getObject(bucketParams).promise();
       const csvReadStream = new stream.Readable();
 
-      csvReadStream._read = () => {};
+      csvReadStream._read = () => {
+      };
       csvReadStream.push(s3object.Body);
 
       await csvReadStream
-        .pipe(csv({headers: true}))
+        .pipe(csv())
         .on('data', async (data) => {
-          results.push(data);
-        })
+          sqsParams.MessageAttributes.Count.StringValue = data.count ? 'empty' : 'stock';
+          sqsParams.MessageBody = JSON.stringify(data);
+          sqs.sendMessage(sqsParams, (err, data) => {
```

Comment:
You can use `sendMessage(sqsParams).promise()`, and then there's no need for outdated "error first callback" feature.